docs/course_materials/exercises/01_Linux/class_exercise_3.md

```diff
@@ -1,9 +1,10 @@
 ### Let's Practice Together \#3
 
 - Open a terminal
-- Navigate to ***test/tmp*** directory in your ***home*** directory
-- Create two files ***a.txt*** and ***b.txt*** in ***tmp*** directory
-- Copy the file ***a.txt*** to the ***test*** directory (one level up directory)
-- Change the name of ***b.txt*** file to ***c.txt***
-- Edit the file ***c.txt*** and write there your name
-- Print the content of the file ***c.txt***
+- Create a Directory **linux3** in you home directory
+- Change Directory  to ***linux3***  that is in your ***home*** directory
```

Comment:
that is in your ***home*** directory - I think it can be removed

Reply:
done